22_kata.py

```diff
@@ -0,0 +1,32 @@
+# https://www.codewars.com/kata/52223df9e8f98c7aa7000062/train/python
+
+
+def rot13(message):
+    chr_range = {
+        "lower": (97, 122),
+        "upper": (65, 90)
+    }
+
+    coded_sentence = ""
+
+    for char in message:
+        if char.isalpha():
+            if char.isupper():
+                char_range = chr_range["upper"]
+            else:
+                char_range = chr_range["lower"]
+
+            unicode_char = ord(char)
+
+            if unicode_char + 13 > char_range[1]:
+                unicode_char = char_range[0] + \
+                    (unicode_char + 12) - char_range[1]
```

Comment:
Did VSC format this line like that? It doesn't seem to be super big and hence I'm surprised why it got broke into two lines :)

Reply:
Yep, during saving moved it down. I don't fight with it. Btw 79 char at line it is common used or it is dead law?